woocommerce/google-listings-and-ads
# Fix E2E test when using RC

### Changes proposed in this Pull Request:

When testing E2E for WC 8.9.0-rc.1 I saw the E2E tests were failing because the workflow was not able to deactivate WooCommerce. 

After investigating. Seems like this is happening because the way we load and install WooCommerce in our wp-env. This PR updates the logic to install WC in the `test-env-setup.sh` file.

_Replace this with a good description of your changes & reasoning._


### Screenshots:

Before
<img width="1118" alt="Screenshot 2024-05-08 at 18 25 26" src="https://github.com/woocommerce/google-listings-and-ads/assets/5908855/9355ce32-3054-4638-8494-603501a2e840">

After
<img width="1135" alt="Screenshot 2024-05-08 at 18 25 51" src="https://github.com/woocommerce/google-listings-and-ads/assets/5908855/0e21b269-a9f4-456b-bb14-eeb1a506d086">


### Detailed test instructions:


1. Don't checkout this PR
2. Run `npm run wp-env:up`
3. Run `npm run -- wp-env run tests-cli -- wp plugin update woocommerce --version=8.9.0-rc.1`
4. See the operation failing. 
5. Checkout this PR
6. Run `npm run wp-env:up`
7. Run `npm run -- wp-env run tests-cli -- wp plugin update woocommerce --version=8.9.0-rc.1`
8. See the operation succeeding. 
9. See workflow for E2E tests failing IN THE SETUP before this PR https://github.com/woocommerce/google-listings-and-ads/actions/runs/9001351242
10. See them correct IN THE SETUP when using this PR https://github.com/woocommerce/google-listings-and-ads/actions/runs/9003604947

### Additional details:


### Changelog entry

> Fix - E2E test when using RC
